_Build/_Language/_Language-patches/pt-PT.browser.dtd

```diff
@@ -0,0 +1,219 @@
+     l10n-central. They will be definitely removed when Australis is ready
+     for mozilla-aurora. -->
+
+<!ENTITY navbarCmd.accesskey           "N">
+<!ENTITY addonBarCmd.label             "Barra de extras">
+<!ENTITY addonBarCmd.accesskey         "A">
+<!ENTITY throbberItem.title            "Indicador de atividade">
+<!ENTITY appMenuButton.label           "Menu">
+<!ENTITY appMenuButton.tooltip         "Abrir menu do &brandShortName;">
+<!ENTITY feedButton.label              "Subscrever">
+<!ENTITY feedButton.tooltip            "Subscrever esta página…">
+<!ENTITY bookmarksButton.tooltip       "Mostrar marcadores">
+<!ENTITY historyButton.tooltip         "Mostra as páginas visitadas recentemente">
+<!ENTITY viewTabsOnTop.label           "Separadores em cima">
+<!ENTITY viewTabsOnTop.accesskey       "c">
+<!ENTITY appMenuEdit.label             "Editar">
+<!ENTITY appMenuToolbarLayout.label    "Disposição da barra de ferramentas…">
+<!ENTITY appMenuFind.label             "Localizar…">
+<!ENTITY appMenuUnsorted.label         "Marcadores não organizados">
+<!ENTITY appMenuWebDeveloper.label     "Programador web">
+<!ENTITY appMenuGettingStarted.label   "Iniciação">
+<!ENTITY appMenuSafeMode.label         "Reiniciar com os extras desativados…">
+<!ENTITY appMenuSafeMode.accesskey     "R">
+<!ENTITY cutButton.tooltip             "Cortar">
+<!ENTITY copyButton.tooltip            "Copiar">
+<!ENTITY pasteButton.tooltip           "Colar">
+<!ENTITY zoomOutButton.tooltip         "Ampliar">
+<!ENTITY zoomInButton.tooltip          "Reduzir">
+<!ENTITY zoomControls.label            "Controlos de tamanho">
+<!ENTITY addonBarCloseButton.tooltip   "Fechar barra de extras">
+<!ENTITY toggleAddonBarCmd.key         "/">
+<!ENTITY backForwardItem.title         "Recuar/Avançar">
+<!ENTITY viewBookmarksSidebar.label    "Mostrar na barra lateral">
+<!ENTITY bookmarksItem.title           "Marcadores">
+<!ENTITY openLocationCmd.label         "Abrir localização…">
+<!ENTITY openLocationCmd.accesskey     "l">
+<!ENTITY bookmarksMenuButton.tooltip   "Mostrar os meus marcadores">
+<!ENTITY restoreTabsOnDemand.label     "Não carregar separadores até os selecionar">
+<!ENTITY restoreTabsOnDemand.accesskey "N">
+
+<!-- cyberfox -->
+<!ENTITY tabGroupsButton.tooltip       "Agrupar separadores">
+<!-- end of strings to be removed post-Australis -->
+<!ENTITY mixedContentBlocked.helplink "Saber mais">
+<!-- Purge RAM browser tweak -->
+<!ENTITY minimizeMemoryCache.label "Limpar cache">
+<!ENTITY turnOnCleanRam.label "Ativa o item de menu Limpar cache.">
+<!-- End Purge RAM browser tweak -->
+
+<!-- Restart Browser Tweak -->
+<!ENTITY restartBrowser.label "Reiniciar navegador">
+<!ENTITY restartBrowserButton.label "Reiniciar navegador">
+<!ENTITY turnOnRestart.label "Ativa o item de menu Reiniciar navegador.">
+<!ENTITY cleanFastRestartCache.label "Clear fast restart cache on browser restart.">
+<!ENTITY restartOptions.label "Restart Options">
```

Comment:
Restart Options as in a group of options for the restart feature, Here is a image
![c5rv6byhunjimko](https://cloud.githubusercontent.com/assets/2867549/20858493/1c208970-b991-11e6-81f9-2f182e251f31.JPG)